environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -2642,7 +2642,7 @@ parameters:
 - name: THANOS_RECEIVE_CONTROLLER_IMAGE
   value: quay.io/observatorium/thanos-receive-controller
 - name: THANOS_RECEIVE_CONTROLLER_IMAGE_TAG
-  value: master-2019-10-18-d55fee2
```

Comment:
This is for the receive controller, not receive itself but this tag doesn't exist for the controller

Reply:
I need to read things fully :facepalm: